packages/store/addon/-private/ts-interfaces/minimum-serializer-interface.ts

```diff
@@ -0,0 +1,280 @@
+/**
+  ## Overview
+
+  In order to properly manage and present your data, `EmberData`
+  needs to understand the structure of data it receives.
+
+  `Serializers` convert data between the format used by an API
+  and the format `EmberData` understands.
+
+  Data received from an API response is `"normalized"` into 
+  [JSON:API](https://jsonapi.org/) (the format used internally
+  by `EmberData`), while data sent to an API is `"serialized"`
+  into the format the API expects.
+
+  ### Implementing a Serializer
+
+  There are only two required serializer methods, one for
+  normalizing data in `JSON:API`, and another for serializing
+  records via `Snapshot`s into the expected API.
+
+  To implement a serializer, export a class implementing the
+  [MinimumSerializerInterface](MinimumSerializerInterface) from
+  the `app/serializers/` directory. An example is below.
+
+  ```ts
+  import EmberObject from '@ember/object';
+
+  export default class ApplicationSerializer extends EmberObject {
+    normalizeResponse(_, __, rawPayload) {
```

Comment:
Maybe it makes sense to leave the actual names for the first two args?

Reply:
Unsure, will think on it

Reply:
Added them